mapactionpy_arcmap/arcmap_runner.py

```diff
@@ -41,34 +44,31 @@ def is_valid_directory(parser, arg):
         parser.error("The directory %s does not exist!" % arg)
         return False
 
+
 def getTemplate(orientation, cookbookFile, crashMoveFolder, productName):
```

Comment:
`getTemplate` should be `get_template`. Also the names of the parameters shouldn't be in camelcase.
See later for comments on handling template names in general.
Also `getTemplate` seems to do two things. Identify the path to the relevant template file AND generate the path to the output MXD. Could/should this be split into two methods?